### DISCUSSION TITLE
Difficult to pull request on github

### DISCUSSION THREAD

**Discussion by Nurudeen010** [community/community#57057] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I just pushed my local repo to my GitHub but not reflecting yet on GitHub. It is requesting for pull request.
Now I am on it, after compare the request, there is no button to press to confirm the pull request.
It is a dead end.
Please help

**Comment by neutrinomuon** [comment]
Hi @Nurudeen010 ,

I hope you're doing well. If you have pushed your local repository to GitHub but the changes are not reflecting yet and you cannot find a button to confirm the pull request, there could be a few possible reasons. 

Here are some tips you can **quickly check**:

1. Double-check the branch pull request: ensure that you are on the correct branch in your local repository and that you are comparing it to the correct branch on GitHub. Sometimes you may have distinct branches in one repository...

2. Repository permissions: Verify that you have the necessary permission to create a pull request in the repository. If you are not the repository owner or a collaborator, you may need to request access or ask to be a collaborator with the owner's permission.

3. Branch visibility: Check if the branch you are trying to create a pull request from is visible on the GitHub repository. Sometimes, branches can be set as private or protected, which can prevent pull requests from being created.

3. Clear cache and refresh: It's possible that your browser is not displaying the latest changes due to cached data. Have you tried clearing your browser cache and refreshing the page to see if the button appears?

-------------

Ok, now in some cases, the repository can be configured to require a reviewer for pull requests, then the button to confirm the pull request will not be available until a reviewer has been assigned and they have approved the changes. If the repository has this requirement, you can follow the next steps to request a review and proceed with the pull request:

- i) After the creation of the pull request, make sure you are on the "Pull Request" tab in the GitHub repository;

- ii) Go to the "Reviewers" section;

- iii) Click on the "Reviewers" input field and start typing the name of the person you want to assign as a reviewer. GitHub will suggest users based on the entered name or GitHub username. However, if you are not the owner of the repository, you may not have the authority to choose a reviewer for the pull request. On the other hand, if you're the owner of the repository, you can assign yourself as the reviewer for your own pull request.;

- iv) Select the appropriate reviewer from the suggestions;

- v) Once the reviewer is selected, they will be notified about the pull request and can review the changes;

- vi) After the reviewer has reviewed and approved the pull request, the "Merge" button will become available, and you can proceed with merging the changes into the branch;

**NOTE:** the availability of the "Merge" button depends on the repository's settings and the reviewer's approval. If the reviewer requires changes or has concerns, they may leave comments or request modifications before approving the pull request.

If you have followed these steps and are still unable to find the button or encounter any issues, I recommend contacting the repository owner or the designated reviewers for further guidance and assistance. 

As a last resort, contact GitHub support: If none of the above steps works or if you believe there might be an issue with the GitHub platform itself, it's best to reach out to GitHub support for assistance. They can investigate the issue and provide specific guidance.

I hope this helps. Please let me know

**Comment by Nurudeen010** [comment]
It does, thanks

**Reply by neutrinomuon** [reply]
Great to know. What was happening?
